app.rb

```diff
@@ -1,8 +1,63 @@
 require 'sinatra/base'
+require './lib/player'
+require './lib/game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+
+  enable :sessions
+
+  before do
+    @game = Game.instance
+  end
+
+  get '/' do
+    erb :index
+  end
+
+  post '/register' do
+    @gametype = params[:gametype]
+    @gamevariant = params[:gamevariant]
+    erb :register
+  end
+
+  post '/register-process' do
+    gametype = params[:gametype]
+    gamevariant = params[:gamevariant]
+    player1 = Player.new(params[:name], "human")
+    if gametype == "Single Player"
```

Comment:
Is there a way to extract this logic out of the controller and call a method instead? I'm not exactly sure how, but it would make this easier to read